src/com/sap/piper/Version.groovy

```diff
@@ -2,13 +2,26 @@ package com.sap.piper
 
 import hudson.AbortException
 
+/**
+ * Version
+ * Handles version numbers.
+ */
 
 class Version implements Serializable {
 
     final def major
     final def minor
     final def patch
 
+    /**
+     * Constructor Version(major, minor, patch)
```

Comment:
Constructs a new Version by copying the arguments.

Reply:
Documentation was removed from the code again.